### ISSUE TITLE
Temperature Not Accurately reporting

### ISSUE THREAD

**Issue by warllo54** [cgarwood/homeassistant-zwave_mqtt#80]
- Home Assistant version: 108.0, was also occurring on 107
- ozwdaemon version or add-on version: latest docker image, I'm not able to find the actual number anywhere.
- homeassistant-zwave_mqtt version: v0.0.9

[mqtt_dump.txt](https://github.com/cgarwood/homeassistant-zwave_mqtt/files/4450981/mqtt_dump.txt)

  I have a Zooz Zwave 4 in 1 sensors model zse40. The temperature reports when manually triggering an update by pressing a button on the sensor and reports an accurate temp of 73 degress F,  however within 30-120 seconds of the device going into sleep mode the temperature reverts to 166.44 degrees F. The device will report other attributes without a button update but no the temp. This was not happening with the old Home Assistant Zwave integrations. I understand this is pre-realease and there will be bugs but I wanted to bring it to your attention.

**Comment by Mystichrome** [comment]
I've noticed similar behavior with this sensor and the aeotec zw100.  However, what I noticed was that the temperature was correct when viewed directly with ozw-admin.  For me it appeared that home assistant was treating the fahrenheit temperature it was receiving from ozwdaemon as if it were celsius and performing the conversion to convert it to fahrenheit.  resolved this by forcing the sensor to fahrenheit in customizations in HA.  After that, my temp sensors for those devices in HA matched the fahrenheit temp seen in ozw-admin.   Maybe your issue is different, but it's something to check.
